Caculator/app/src/main/java/com/example/caculator/ultils/StringUltils.kt

```diff
@@ -0,0 +1,94 @@
+package com.example.caculator.ultils
+
+import java.text.DecimalFormat
+import java.text.DecimalFormatSymbols
+import java.util.*
+
+class StringUltils {
+    companion object{
```

Comment:
Có thể tìm hiểu về từ khóa object trong kotlin 
```suggestion
object StringUltils {
```